plugins/content/loadmodule/loadmodule.php

```diff
@@ -129,8 +129,15 @@ public function onContentPrepare($context, &$article, &$params, $page = 0)
 		{
 			foreach ($matchesmodid as $match)
 			{
-				$id     = trim($match[1]);
-				$output = $this->_loadid($id);
+				$matchesidlist = explode(',', $match[1]);
+				$id     = trim($matchesidlist[0]);
+				// We may not have a module style so fall back to the plugin default.
+				if (!array_key_exists(1, $matchesidlist))
+				{
+					$matchesidlist[1] = $style;
+				}
```

Comment:
Add empty line after
```suggestion
				}

```